src/main/java/org/openrewrite/java/testing/mockito/AnyStringToNullable.java

```diff
@@ -15,24 +15,20 @@
  */
 package org.openrewrite.java.testing.mockito;
 
-import org.openrewrite.ExecutionContext;
-import org.openrewrite.Recipe;
-import org.openrewrite.TreeVisitor;
+import org.openrewrite.*;
 import org.openrewrite.internal.lang.Nullable;
-import org.openrewrite.java.JavaIsoVisitor;
-import org.openrewrite.java.JavaTemplate;
-import org.openrewrite.java.MethodMatcher;
+import org.openrewrite.java.*;
 import org.openrewrite.java.search.UsesMethod;
 import org.openrewrite.java.tree.J;
 
 import java.time.Duration;
 
 /**
- * S
+ * Replace Mockito 1.x `anyString()` with `nullable(String.class)`
  */
 public class AnyStringToNullable extends Recipe {
-
-    public static final String ARGUMENT_MATCHERS_ANY_STRING = "org.mockito.Mockito anyString()";
+    private static final MethodMatcher ANY_STRING = new MethodMatcher("org.mockito.ArgumentMatchers anyString()");
+    private static final String MOCKITO_CLASS_PATH = "mockito-core-3.12.4";
```

Comment:
This recipe is intended to be used only when migrating from Mockito 1.x to 2.10 and above, as that's when the behavior surrounding the nullability of `anyString()/any(Class)` changed. We do not want to migrate usages of `anyString()/any(Class)` from Mockito 3+ to `nullable(Class)`, as that would change the semantics again.

Given that context, would you then agree with the following change?
```suggestion
    private static final MethodMatcher ANY_STRING = new MethodMatcher("org.mockito.Mockito anyString()");
    private static final String MOCKITO_CLASS_PATH = "mockito-all-1.10.19";
```

I think since we match on `Mockito` rather than `ArgumentMatchers`, newer code should be reasonable safe. Would you agree to that as well?